app/controllers/application_controller.rb

```diff
@@ -1,3 +1,14 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  before_action :set_raven_context
+
+  private
+
+  def set_raven_context
+    Raven.user_context(name: session[:beta_user])
```

Comment:
What does :beta_user contain? I can't find where in the code we add to that session

Reply:
https://github.com/datagovuk/find_data_beta/blob/master/app/controllers/logged_area_controller.rb#L21